tasks/oah-update.yml

```diff
@@ -0,0 +1,13 @@
+---
+
+- name: Uninstall the existing version of Docker
+  apt:
+    name: "{{ docker_uninstall_items}}"
+    state: absent
+    update_cache: yes
```

Comment:
Fails while updating cache. Either turn it off or add ignore errors

Reply:
Adding ignore_error if it is a local caching update issue. Please mention the issue if it's something related to playbook.